packages/manager/modules/billing/src/components/cancel-commitment/cancel-commitment.controller.js

```diff
@@ -6,6 +6,7 @@ export default class ServicesActionsCtrl {
   }
 
   cancelCommitment() {
+    this.trackClick();
```

Comment:
What will happen if no `trackClick` defined (`trackClick` value will be `undefined`) ?

```suggestion
    this.trackClick && this.trackClick();
```

Reply:
Code suggestion implemented. Thanks!